projet/projects/serializers.py

```diff
@@ -105,3 +116,17 @@ class Meta:
             "over_budget_by",
             "over_budget_by_percentage",
         )
+
+
+class PersonalBudgetSerializer(serializers.ModelSerializer):
```

Comment:
We create simple serializers for both the personal expense model and the personal budget model